src/api/newsApi.ts

```diff
@@ -0,0 +1,20 @@
+import { Category } from '../types/category';
+import { Article } from '../types/article';
+import axios from 'axios';
+
+const API_KEY = import.meta.env.VITE_NEWS_API_KEY;
```

Comment:
구조분해 할당으로도 리팩터링 가능할 것 같네요.

```suggestion
const { VITE_NEWS_API_KEY } = import.meta.env;
```

Reply:
훨씬 깔끔한 것 같네요! 수정했습니다!